html/webappapis/scripting/processing-model-2/compile-error-data-url.html

```diff
@@ -17,11 +17,11 @@
         ran = true;
         col_value = d;
         assert_equals(typeof a, 'string', 'first arg');
-        assert_equals(b, 'data:text/javascript,for(;) {}', 'second arg');
+        assert_equals(b, 'data:text/javascript,for(;){}', 'second arg');
         assert_equals(typeof c, 'number', 'third arg');
     });
   </script>
-  <script src="data:text/javascript,for(;) {}"></script>
```

Comment:
Good catch. AFAICT the `{}` will not be percent-escaped when serializing the URL per https://url.spec.whatwg.org/ so just removing the space sounds good.